libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/atomic/molecules/ComposerAlertMolecule.kt

```diff
@@ -78,7 +78,11 @@ fun ComposerAlertMolecule(
                         text = content,
                         modifier = Modifier.weight(1f),
                         style = ElementTheme.typography.fontBodyMdRegular,
-                        color = ElementTheme.colors.textPrimary,
+                        color = if (isCritical) {
+                            ElementTheme.colors.textCriticalPrimary
```

Comment:
It's a bit "red on red" in dark theme, is it the expected design?

<img width="327" alt="image" src="https://github.com/user-attachments/assets/4ae0a172-7264-4fed-afdd-ac4648621bee" />

Reply:
There is no dark version in figma. I am color blind so not sure if it's strange or not ^^